openedx/features/job_board/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from django_countries.fields import CountryField
+from model_utils.models import TimeStampedModel
+
+from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES
+
+
+class Job(TimeStampedModel):
+    """
+    This model contains all the fields related to a job being
+    posted on the job board.
+    """
+    title = models.CharField(max_length=255)
+    company = models.CharField(max_length=255)
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES)
+    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES)
+    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES)
+    city = models.CharField(max_length=255, blank=True, null=True)
+    country = CountryField(null=True, blank=True)
+    description = models.TextField()
+    function = models.TextField()
+    responsibilities = models.TextField()
+    website_link = models.URLField(max_length=255, blank=True, null=True)
+    contact_email = models.EmailField(max_length=255, blank=True, null=True)
+    logo = models.ImageField(upload_to="job_board/uploaded_logos/", blank=True, null=True)
```

Comment:
where did we get this from?
`"job_board/uploaded_logos/"`
can we store it in a constant?

Reply:
This will create a directory structure in `MEDIA_ROOT` and upload the logos there. Also, since it's only being used here, I don't see what value storing it in a constant bring?

Reply:
single quotes please 😋

Reply:
Done, @zeshanarifios.